src/components/ToolIcon.scss

```diff
@@ -181,6 +183,17 @@
     }
   }
 
+  .TooltipIcon {
+    width: 0.9em;
+    height: 0.9em;
+    margin-left: 5px;
+    margin-top: 1px;
+
```

Comment:
Maybe `cursor: help;`

Reply:
Nice suggestion.

Reply:
It would look like this
![Peek 2020-12-14 03-27](https://user-images.githubusercontent.com/24036721/102025277-d7498e00-3dbc-11eb-8471-3de197a03c27.gif)

Reply:
Doesn't look good.. maybe skip it :)